app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,152 @@
+/* Global Styles */
+body {
+  background-color: #867373;
+  font-family: 'Arial', sans-serif;
+  font-size: 12px;
+}
+
+h3 {
+  margin: 0;
+  font-size: 1.2rem;
+}
+
+ul {
+  list-style: none;
+  padding: 1rem;
+}
+
+a {
+  color: black;
+  text-decoration: none;
+  font-size: 1rem;
+}
+
+a:visited {
+  color: inherit;
+}
+
+/* Form Styles */
+.input {
+  padding: 2%;
+  border-radius: 5px;
+  margin-bottom: 10px;
+  margin-left: 10px;
+}
+
+.f-label {
+  display: flex;
+  align-items: flex-start;
+}
+
+/* Button Styles */
+.btn-container {
+  display: flex;
+  justify-content: center;
+}
+
+.btn {
+  font-size: 1.5rem;
+  padding: 10px 15px;
+  border-radius: 5px;
+  background-color: #ccc;
+  margin-top: 1rem;
+  color: rgb(0, 30, 255);
+  cursor: pointer;
+}
+
+.center-btn {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.flex-right {
+  display: flex;
+  flex-direction: row-reverse;
+}
+
+.form-heading {
+  text-align: center;
+  font-weight: bold;
+  font-size: 1.7rem;
+  padding: 0.5rem 0;
+}
+
+#error_explanation {
+  color: #ff3b30;
+}
+
+.col-md-4 {
+  margin-left: 16%;
+  margin-top: 5rem;
+}
+
+.email_input,
+.pass_input {
+  width: 100%;
+  margin-bottom: 1rem;
+  background: white;
+  color: black;
+  padding: 5px;
+  border: 1px solid black;
+  border-radius: 5px;
+}
+
+.email_input::placeholder,
+.pass_input::placeholder {
+  color: black;
+  font-weight: bold;
+}
+
+.form-check-label {
+  font-size: 1rem;
+  color: black;
+}
+
+.btn-primary {
+  width: 100%;
+  height: 2.5rem;
+  border: none;
+  font-size: 1.2rem;
+  background: blue;
+  color: black;
+  border-radius: 10px;
+}
+
+a:hover {
+  border-color: #e74c3c;
+  color: #e74c3c;
+}
+
+.lik-sec {
+  margin-top: 2rem;
+}
```

Comment:
- [ ] [OPTIONAL] Another great job yet again! But you could also make this a killer card by perfecting the user experience and aesthetics on this sign-up and sign-in form. Currently, the width is as long as 100% of the available viewport. I think it would be a lot nicer if its reduced to say 40% and placed at the center.

![image](https://github.com/leloufadel/Recip-App/assets/99715640/e7b93c01-bb67-4054-8543-226c6b533151)